test/NuGet.Core.Tests/NuGet.Protocol.Core.v3.Tests/PackageMetadataResourceV2FeedTests.cs

```diff
@@ -17,11 +17,14 @@ public class PackageMetadataResourceV2FeedTests
         public async Task PackageMetadataResource_Basic()
         {
             // Arrange
+            var serviceAddress = TestUtility.CreateServiceAddress();
```

Comment:
Spoke offline. This should ideally inject the `HttpSource` cache directory somehow to be a `TestDirectory` (which is existing test infrastructure that cleans up itself). This can be done with a new resource, à la `HttpSourceCacheResource`. Could you file an issue to this effect? Low priority, methinks.